src/test/java/com/vinylteam/vinyl/service/impl/DefaultUserServiceTest.java

```diff
@@ -173,30 +183,54 @@ void signInCheckNonExistingUserTest() {
         //prepare
         String newEmail = "user2@wax-deals.com";
         String newPassword = "password2";
-        when(mockedUserDao.getByEmail(newEmail)).thenReturn(Optional.empty());
+        when(mockedUserDao.findByEmail(newEmail)).thenReturn(Optional.empty());
         //when
         Optional<User> actualOptional = userService.signInCheck(newEmail, newPassword);
         //then
         assertTrue(actualOptional.isEmpty());
-        verify(mockedUserDao).getByEmail(newEmail);
+        verify(mockedUserDao).findByEmail(newEmail);
         verify(mockedSecurityService, never()).checkPasswordAgainstUserPassword(eq(null), eq(newPassword.toCharArray()));
     }
 
+    @Test
+    @DisplayName("Checks if .signInCheck(...) with existent user's email and wrong password as arguments returns Optional.empty()," +
+            " userDao.getByEmail(...) and securityService.checkPasswordAgainstUserPassword(...) are called")
+    void signInCheckExistingNotVerifiedUserWrongPasswordTest() {
+        //prepare
+        String existingEmail = "user1@wax-deals.com";
+        String rightPassword = "password1";
+        Optional<User> optionalUserFromDB = Optional.of(users.get(0));
+        ConfirmationToken confirmationToken = tokens.get(0);
+        optionalUserFromDB.get().setId(1L);
+        optionalUserFromDB.get().setStatus(false);
```

Comment:
Would it be better firstly to set values for user and only then put it into Optional wrapper?